compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/types/TupleTypeNode.java

```diff
@@ -30,6 +31,6 @@
  */
 public interface TupleTypeNode {
 
-    List<? extends TypeNode> getMemberTypeNodes();
+    List<? extends BLangSimpleVariable> getMemberTypeNodes();
```

Comment:
`getMemberTypeNodes` if required can still be present and should return the types.

This should be something like `getMemberNodes`.

Please check and fix wherever applicable.